src/DataStructures.jl

```diff
@@ -33,8 +33,8 @@ module DataStructures
     export IntDisjointSets, DisjointSets, num_groups, find_root, in_same_set, root_union!
 
     export AbstractHeap, compare, extract_all!
-    export BinaryHeap, binary_minheap, binary_maxheap, nlargest, nsmallest
-    export MutableBinaryHeap, mutable_binary_minheap, mutable_binary_maxheap
+    export BinaryHeap, BinaryMinHeap, BinaryMaxHeap, nlargest, nsmallest
+    export MutableBinaryHeap, MutableBinaryMinHeap, MutableBinaryMaxHeap
```

Comment:
still need to export the old deprecated names.
Til removed.

but can do so on own line, making it easy to remove them later.

Reply:
The `@deprecate` macro exports the functions, so this should be OK.

Reply:
Cool. I learn something new every day.